### DISCUSSION TITLE
Add @kayrus to maintainers

### DISCUSSION THREAD

**Discussion by EmilienM** [gophercloud/gophercloud#2694] [General]
@kayrus has been very involved in Gophercloud for a long time and has provided valuable contributions. I propose that we add him to the maintainers.

**Comment by EmilienM** [comment]
cc @gophercloud/core

**Comment by mandre** [comment]
Yes, I'd be very much in favor of adding @kayrus to the maintainers team. He has a long history of high quality PRs and has shown interest helping with the maintenance of gophercloud.

**Comment by EmilienM** [comment]
Positive feedback on Slack & here, I'll add you today without waiting.
Thanks for all your work!

**Reply by kayrus** [reply]
Thank you!
